raiden/event_handler.py

```diff
@@ -245,6 +245,15 @@ def handle_channelnew(self, state_change):
                 token_address,
                 channel_address,
             )
+            # This is little hacky, used when bootstraping a token network.
+            # See ConnectionManager for details.
+            have_bootstrap = ((participant1 == connection_manager.BOOTSTRAP_ADDR) or
+                              (participant2 == connection_manager.BOOTSTRAP_ADDR))
+            if not have_bootstrap:
+                self.raiden.start_health_check_for(
+                    participant2 if
+                    participant1 == self.raiden.address else
+                    participant1)
```

Comment:
Are you sure about not registering the bootstrap channel? I thought it was neccessary to have at least one channel registered when creating a new token network, so other participants will discover creator's address.